front/components/agent_builder/MCPServerViewsContext.tsx

```diff
@@ -1,24 +1,108 @@
+import { groupBy } from "lodash";
```

Comment:
Don't really need to be reviewed, this is to copy the logic to group MCPServerViews